### ISSUE TITLE
Have a schedule for the proofing/baking/resting in a single function

### ISSUE THREAD

**Issue by tpoisot** [PoisotLab/Omnomnomivores#13]
The plan here is to have a series of generations with set values (proofing); then we do the increase in temperature (baking); and then some number of generations with the final values (resting). This should be handled by a single function, and we can simply use the baking update schedule to manifest the environmental values at each timestep. The switch between logistic and linear can be handled by passing a function as an argument somewhere.

https://github.com/PoisotLab/Omnomnomivores/blob/df59c4ce259b354157d9b012f3bdfc4d263c2e45/01_code.jl#L178

**Comment by TanyaS08** [comment]
Do you think we need the logistics and linear? Past Tanya kept the relic linear change because sentimentality reasons I guess. Obvi not opposed to having the optionality (thats a word?) for both but we can also just choose to keep it simple and light

**Comment by TanyaS08** [comment]
Okay so I looked at the source code and I think this will actually be really simple to implement and we can then just build one massive community matrix that we then subdivide into the different phases of baking a binary species landscape. And by subdivide I mean index appropriately

**Comment by tpoisot** [comment]
Yup that's what I'm going to do (tomorrow) -- I would like to keep a function to do the baking steps, because then someone may play with sines, or other arbitrary temporal dynamics.
